DZ1_Example.py

```diff
@@ -0,0 +1,96 @@
+# 1 Задание
+
+class MyTime:
+    def __init__(self, *args):
+        if len(args) == 0:
+            self.hours = 0
+            self.minutes = 0
+            self.seconds = 0
+        elif len(args) == 1 and isinstance(args[0], str):
+            h, m, s = map(int, args[0].split(':'))
+            self.hours = h
+            self.minutes = m
+            self.seconds = s
+        elif len(args) == 3:
+            self.hours = args[0]
+            self.minutes = args[1]
+            self.seconds = args[2]
+        elif len(args) == 1 and isinstance(args[0], MyTime):
+            self.hours = args[0].hours
+            self.minutes = args[0].minutes
+            self.seconds = args[0].seconds
+
+        self.show_time()
+
+    def print_args(self):
+        print(self.hours, self.minutes, self.seconds)
+
+    def show_time(self):
```

Comment:
This method, in theory, should not be called show_time, but something like normalize_time, since it does not display the time, but converts the time/normalizes it